lib/mail/elements/content_location_element.rb

```diff
@@ -1,21 +1,22 @@
 # encoding: utf-8
+# frozen_string_literal: true
 module Mail
   class ContentLocationElement # :nodoc:
-    
+
     include Mail::Utilities
-    
+
     def initialize( string )
       content_location = Mail::Parsers::ContentLocationParser.new.parse(string)
       @location = content_location.location
     end
-    
+
     def location
       @location
     end
-    
+
     def to_s(*args)
       location.to_s
     end
-    
+
```

Comment:
The change removes the superfluous whitespace that was in the file (my editor did that I guess) -- if you're saying the line is unneeded, it was there before but with some spaces at the beginning